src/Sentry/Laravel/SentryHandler.php

```diff
@@ -90,7 +90,9 @@ function ($record) use ($level) {
         $record = array_reduce(
             $records,
             function ($highest, $record) {
-                if ($record['level'] > $highest['level']) {
+                if (!isset($highest['level'])
+                    || $record['level'] > $highest['level']
+                ) {
```

Comment:
In your original report (#514) you mention `$highest` is `null` so why not check for that instead of checking if the `level` key is missing since it should never miss on a Monolog record IIRC.

```suggestion
                if ($highest === null || $record['level'] > $highest['level']) {
```

Reply:
Good point. That would make the code cleaner and signal the issue it addresses more clearly. I've changed it.